File: submissions/sm_037_srikanth/week_03/day_3/session_2/count_vowels.js
var string='Masai School';
function countVowels(string){
    var vowels=0;
    for (i=0;i<=string.length;i++){
        if(string[i] == 'o'||string[i]=='e'||string[i]=='u'||string[i]=='i'||string[i]=='a'){
            vowels=vowels+1;
        }
    }
     return console.log(vowels);
}
countVowels(string);

File: submissions/sm_110_mihir/week_14/day_3/count_vowels.py
lower=['a','e','i','o','u']
count_lower=0
upper=['A','E','I','O','U']
count_upper=0
string="MASAI School"
for char in string:
    if char in lower:
        count_lower+=1
    elif char in upper:
        count_upper+=1
print("lower - ",str(count_lower))
print("UPPER - ",str(count_upper))